### PR TITLE
added missing quotes for IAM binding

### PR DESCRIPTION
**Pull request by Techdox**
The IAM binding for the Workload Identity is missing the quotes around the service account, if you don't have these quotes it will throw up an error about the brackets "[velero/velero]".  See GCP documentation https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#using_from_your_code

### CODE REVIEW

**Review by carlisia — APPROVED** [review]
Thank you for this.

I'm approving it, but you'll need to sign your commit before it can be merged. Thank you.

**Comment by Techdox** [comment]
> Thank you for this.
> 
> I'm approving it, but you'll need to sign your commit before it can be merged. Thank you.

Thanks! I have signed my commit, took we awhile to figure out what email address I initially used for the commit :D

**Review by carlisia — APPROVED** [review]

**Review by jenting — APPROVED** [review]

**Comment by carlisia** [comment]
@jenting I added you as maintainer in this and the other plugin repos, please re-approve when you have a chance. Thank you.

**Review by jenting — APPROVED** [review]

**Review by jenting — APPROVED** [review]

**Pull request merged**
